docs/sdks/client-sdks/react-native/quickstart.mdx

```diff
@@ -0,0 +1,231 @@
+---
+title: Quickstart
+sidebar_position: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The Eppo React Native SDK lets you manage feature flags and experiments in your react native applications 
+
+The SDK handles all the complexity of feature flag evaluation and experiment assignment locally in your application, with no network calls required after initial setup. This guide will walk you through installing the SDK and implementing your first feature flag, experiment, and contextual bandit.
+
+## Installation
+
+First, install the SDK into your current project.
+
+The Eppo React Native SDK is available via a number of package managers or via a CDN. Use whichever one you prefer.
+
+<Tabs>
+<TabItem value="yarn" label="Yarn">
+
+```bash
+yarn add @eppo/react-native-sdk
+```
+
+</TabItem>
+
+<TabItem value="npm" label="NPM">
+
+```bash
+npm install @eppo/react-native-sdk
+```
+
+</TabItem>
+
+</Tabs>
+
+Depending on whether you are running a feature flag, an experiment, or a bandit, the initialization code will be slightly different.
+
+## Feature Flags
+Feature flags are a way to toggle features on and off without needing to deploy code.
+
+### Initialize the SDK
+
+[Create an SDK key](/sdks/sdk-keys) if you don't already have one. 
+
+First, initialize the Eppo SDK at the app root using your SDK key. The SDK key is different from the project API key. 
+
+```js
+import init from "@eppo/react-native-sdk";
+
+await init({apiKey: 'SDK_KEY'});
+``` 
+
+:::note
+The SDK key is different from the project API key. You can find your SDK key in the [SDK Keys section of the Eppo interface](https://eppo.cloud/configuration/environments/keys).
+:::
+
+### Assign a variant
+
+Once initialized, the SDK uses a singleton pattern, making it accessible from anywhere in your application. To make assignments, import the SDK and use `getInstance()` to access the SDK instance.
+
+### How assignments work
+
+The SDK periodically retrieves configuration rules from the Eppo server that define how subjects should be allocated to variants. When you call an assignment function, the SDK evaluates these rules locally without making additional network requests.
+
+Each assignment requires:
+
+- **Flag Key**: Identifies which set of configuration rules to use
+- **Subject Key**: A unique identifier for the subject (usually a user ID)
+- **Subject Attributes**: Optional key-value pairs containing additional information used for rule evaluation
+- **Default Value**: Fallback value if assignment fails or rules don't match
+
+```js
+import * as EppoSdk from "@eppo/react-native-sdk";
+
+// Get SDK instance
+const eppoClient = EppoSdk.getInstance();
+
+// Define assignment parameters
+const featureKey = "my-neat-feature";
+const subjectKey = getCurrentUser() || "anonymous";
+const subjectAttributes = {
+    'country': user.country,
+    'device': user.device,
```

Comment:
seems odd we have `user` here but `getCurrentUser()` above. 

I'd replace `getCurrentUser()` with `user?.id` and make these optionally chained (`user?.xxxxxx`) as well

Also we should be consistent with single or double quotes, probably conform on double quotes